src/features/canvass/components/LocationDialog/index.tsx

```diff
@@ -99,6 +105,11 @@ const LocationDialog: FC<LocationDialogProps> = ({
   const [selectedHouseholdId, setSelectedHouseholdId] = useState<number | null>(
     null
   );
+  const [selectedHouseholsdIds, setSelectedHouseholdsIds] = useState<number[]>(
```

Comment:
The variable name here is misspelled (and pluralized wrong IMO).

```suggestion
  const [selectedHouseholdIds, setSelectedHouseholdIds] = useState<number[]>(
```